packages/shared/tailwind.config.js

```diff
@@ -109,7 +109,7 @@ module.exports = {
                 100: '#FFF4DF',
                 200: '#FFE9C0',
                 300: '#FFDFA0',
-                400: '#0F4CEB',
+                400: '#FFD481',
```

Comment:
in the UI kit we have #0F4CEB here, where did you take this color from?

Reply:
In Figma, when you click on the element that represents the color yellow-400 you can check in the inspect tab its CSS properties, the background color for the yellow-400 is #FFD481